LibreNMS/OS/Coriant.php

```diff
@@ -25,9 +25,11 @@
 
 namespace LibreNMS\OS;
 
+use App\Models\TnmsAlarm;
 use App\Models\TnmsneInfo;
 use App\Observers\ModuleModelObserver;
 use LibreNMS\Interfaces\Polling\OSPolling;
+use LibreNMS\Util\ModuleModelObserver;
```

Comment:
A parse error occurred: Cannot use LibreNMS\Util\ModuleModelObserver as ModuleModelObserver because the name is already in use

Reply:
This file does not exist. 

App\Observers\ModuleModelObserver is the only existing one